hourglass.py

```diff
@@ -53,10 +54,13 @@ def load_and_filter_annotations(self, path_to_train_anns,path_to_val_anns, subse
         print(f"Train/Val dfs contains {len(train_df)}/{len(val_df)} anns")
         return train_df.reset_index(), val_df.reset_index()
 
-    def _start_train(self, batch_size, model_base_dir, epochs, initial_epoch, model_subdir, current_time, subset):
+    def _start_train(self, batch_size, model_base_dir, epochs, initial_epoch, model_subdir, current_time, subset, pickle_name):
         self._compile_model()
-
-        train_df, val_df = self.load_and_filter_annotations(DEFAULT_TRAIN_ANNOT_PATH, DEFAULT_VAL_ANNOT_PATH, subset)
+        # insert logic here for pickled dataframes
+        if pickle_name != None:
+            train_df, val_df, test_df = get_pickle(pickle_name)
```

Comment:
Since `test_df` likely won't be used in this method, could change the unpacking to: `train_df, val_df, _`

Reply:
we may need some way to load test (which for us might be the val2017 set) separately than the others, since we'll be running evaluation on the test set. We'll have to decide how many entries to set aside for validation and testing.